static/js/compose.js

```diff
@@ -24,10 +25,13 @@ function make_uploads_relative(content) {
     return content.replace(exports.uploads_re, "]($1)");
 }
 
-function show_all_everyone_warnings() {
+function show_all_everyone_warnings(is_announce_stream = false) {
```

Comment:
With this change, the function name doesn't really reflect the functionality anymore. I'd prefer renaming this function to smt like `show_warnings(warning_type)`, where `warning_type` is a value in {'all', 'everyone', 'announce'}, and then change the function logic and calls respectively.

Reply:
OK, I changed it to `show_warning` in 181a2cd5c30e66ee3245a080283d46667e13ad84.